src/components/fileManaging/containerTestFilesManaging.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
+import FileContainer from './filesContainer';
+
+//State and Props
+interface IProps {
+
+}
+
+interface IState {
+  attachments: Array<IAttachement>
+}
+
+interface IAttachement {
+  id?: number,
+  attach_type: string,
+  key: string,
+  fileName: string
+  file?: any
+}
+
+class FakeContainerFiles extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      attachments: [{ id: 1, attach_type: 'cv', key: '1234.pdf', fileName: 'fileNameExemple' }]
```

Comment:
I think it would be better to wait for the whole candidature form to be completed before merging this. Thus, We could use real attachment here. What do you think ?

Reply:
Yes! good idea